docs/en/04-supported-codebases/mmcls.md

```diff
@@ -1,20 +1,149 @@
-# MMClassification Support
+# MMClassification Deployment
```

Comment:
It would be better to show an overview at the beginning.
- [Installation](#installation)
  - [Install mmcls](#install-mmcls)
  - [Install mmdeploy](#install-mmdeploy)
- [Convert models](#convert-models)

Reply:
catalogue is added